docs/instruments.rst

```diff
@@ -0,0 +1,20 @@
+instruments
+===========
+
+.. dfhack-tool::
+    :summary: Show how to craft instruments.
+    :tags: fort productivity
```

Comment:
since this tool doesn't actually generate workorders, I'd classify it as `inspection` instead of `productivity`

Reply:
edit: maybe copy the tags from `necronomicon`: `fort inspection productivity items`.

Reply:
Hmm, upon some thought, I'd argue that `necronomicon` is the one that's misclassified. At a glance, it seems to be the only tool tagged `productivity` that is not actually modifying gamestate. Also, this tool, unlike `necronomicon`, does not actually "interact with in-game items". So I think your first reaction was the correct one: `fort inspection`.

Reply:
that's a good point. I'll remove `productivity` from `necronomicon`

Reply:
@myk002 I think you moved too fast and merged it tagged `fort productivity` rather than `fort inspection`.

Reply:
I was trying to avoid creating multiple commits in case you disagreed. So I thought you'd change that before merging. :man_shrugging:

Reply:
oop, will fix in situ